fairchive-webapp/src/main/webapp/createDataset.xhtml

```diff
@@ -140,25 +140,35 @@
                                      action="#{CreateDatasetPage.save}"
                                      update="formContent"
                                      onstart="PF('datasetFormBlockUI').show();"
-                                     onerror="PF('saveStatusPoll').start();"
-                                     onsuccess="PF('datasetFormBlockUI').hide();"
-                                     oncomplete="$(document).scrollTop(0);"
+                                     oncomplete="PF('datasetFormBlockUI').show(); PF('saveStatusPoll').start(); $(document).scrollTop(0);"
```

Comment:
When we would have validation error then the flow acts weird.
Page scrolls to top and we have validation error message (this is ok)
But the overlay is still showing up and we have to wait 2 seconds to unblock the overlay.
And after this 2 seconds the "validation error" message dissapears, but it shouldn't.

Maybe we can somehow differentiate when we start the polling `PF('saveStatusPoll').start();` and when there is no save running then we do not start the polling at all?